applications/callflow/src/module/cf_park.erl

```diff
@@ -118,8 +119,11 @@ handle_nomatch_with_empty_referred_to(Data, Call, PresenceType, ParkedCalls, Slo
 
 -spec direct_park(kz_term:ne_binary(), kz_json:object(), kz_json:object(), kz_json:object(), kapps_call:call()) -> 'ok'.
 direct_park(SlotNumber, Slot, ParkedCalls, Data, Call) ->
-    case save_slot(SlotNumber, Slot, ParkedCalls, Call) of
+    MaxSlotNumber = kz_json:get_integer_value(?MAX_SLOT_NUMBER_KEY, Data),
+    case save_slot(SlotNumber, MaxSlotNumber, Slot, ParkedCalls, Call) of
         {'ok', _} -> parked_call(SlotNumber, Slot, Data, Call);
+        {'error', 'max_slots_exceeded'} ->
+            cf_exe:continue(Call);
```

Comment:
use `cf_exe:continue/2` with a key (maybe `MAX_SLOTS` ?) so that next action knows the reason

Reply:
@lazedo great idea! I have added this in the last commit

Reply:
@danielfinke was thinking in a known key like `MAX_SLOTS` not the number by itself

Reply:
@lazedo yup, that's what I did. Added:
`-define(MAX_SLOT_EXCEEDED, 'max_slot_exceeded').`
and `?MAX_SLOT_EXCEEDED` is the key that is used for `cf_exe:continue/2`

Reply:
ah! got fooled by kz_term:to_binary ( `kz_term:to_binary(?MAX_SLOT_EXCEEDED)` ).
can we put this in uppercase, please ? `kz_term:to_upper_binary` instead of `kz_term:to_binary`. most continue keys are in uppercase as the returned values from commands

Reply:
@lazedo OK, I have uppercased the continue key